daskernetes/core.py

```diff
@@ -85,17 +84,15 @@ def __init__(
 
         self.namespace = namespace
         self.name = name
-        self.worker_image = worker_image
-        self.worker_labels = (worker_labels or {}).copy()
-        self.threads_per_worker = threads_per_worker
-        self.env = dict(env)
```

Comment:
If we want to we can continue supporting these kinds of common configurations by modifying the template in the `_make_pod` method.  I don't know if this is desired, but it might be useful if there are parameters that we expect users to want to change?

Reply:
I think I'm going to move the extra_* stuff to the constructor, and just have a 'common configuration' thing as another classmethod.